apps/bajour/components/bajour/best-of-wepublish.tsx

```diff
@@ -0,0 +1,264 @@
+import {styled} from '@mui/material'
+import {BuilderTeaserGridBlockProps, Image as BuilderImage} from '@wepublish/website'
+import Image from 'next/image'
+
+import {NextWepublishLink} from '../should-be-website-builder/next-wepublish-link'
+import babaNews from './peers-logos/baba-news.png'
+
+type BestOfWePublishProps = Omit<BuilderTeaserGridBlockProps, 'teasers'> & {
+  // has to be any as we cannot import CustomTeaser from @wepublish/website/api
+  teasers: any[]
+}
+
+const BestOfWePublishWrapper = styled('div')`
+  margin: 2rem 0;
+  display: grid;
+  grid-gap: 2rem;
+  padding: 0 1rem;
+
+  ${({theme}) => theme.breakpoints.up('md')} {
+    padding: 0;
+  }
+`
+
+const Wrapper = styled('div')`
+  display: grid;
+  gap: ${({theme}) => theme.spacing(2)};
+  grid-template-columns: repeat(2, 1fr);
+  align-items: stretch;
+
+  ${({theme}) => theme.breakpoints.up('md')} {
+    grid-template-columns: repeat(3, 1fr);
+  }
+`
+
+const TeaserBackground = styled(BuilderImage)`
+  width: 100%;
+  object-fit: cover;
+`
+
+const PeerLogoWrapper = styled('div')`
+  position: absolute;
+  top: 0;
+  left: 0;
+  padding-right: 0.4rem;
+  padding-bottom: 0.4rem;
+  background-color: white;
+  z-index: 1;
+  width: 3rem;
+
+  ${({theme}) => theme.breakpoints.up('md')} {
+    width: 4rem;
+    padding-right: 0.6rem;
+    padding-bottom: 0.6rem;
+  }
+`
+
+const LinkWrapper = styled(NextWepublishLink)`
+  display: grid;
+  grid-template-columns: repeat(24, 1fr);
+  grid-template-rows: auto;
+  align-items: center;
+  grid-template-columns: 1fr;
+  spacing: ${({theme}) => theme.spacing(2)};
+  padding-top: 1rem;
+  padding-left: 1rem;
+  position: relative;
+
+  ${({theme}) => theme.breakpoints.up('md')} {
+    padding-top: 2rem;
+    padding-left: 2rem;
+  }
+`
+
+const BestOfWePublishHeader = styled('div')`
+  width: 100%;
+  background-color: #ffbaba;
+
+  > span {
```

Comment:
Don't use direct elements please and ideally avoid nesting unless not really possible.

Reply:
is there a reason that nesting should be avoided? is there some best practice in styled-components/emotion or is it just simply because of specificity?